modules/core/src/main/scala/gem/Observation.scala

```diff
@@ -26,4 +31,12 @@ object Observation {
       fromString(s).getOrElse(sys.error("Malformed Observation.Id: " + s))
 
   }
-}
\ No newline at end of file
+
+  implicit def ObservationTraverse[T]: Traverse[Observation[?]] =
```

Comment:
I guess the `?` comes from the `kind-projector` plugin? Do you know if IDEA accepts that syntax for lambdas?